codespell_lib/data/dictionary.txt

```diff
@@ -24500,6 +24502,8 @@ retalitated->retaliated
 retalitation->retaliation
 retangles->rectangles
 retanslate->retranslate
+retcieve->retrieve, receive,
+retcieved->retrieved, received,
```

Comment:
May I recommend `retciever` and `retcievers`?
```suggestion
retcieved->retrieved, received,
retciever->retriever, receiver,
retcievers->retrievers, receivers,
```

Reply:
Sure, suggestions / recommendations are always appreciated. Thanks a lot for your review, applied these suggestion in b6cb7709e34d513e7b23b011593496d5edb30dfe